packages/suite/src/components/wallet/PrivacyAccount/AnonymityLevelIndicator.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { forwardRef } from 'react';
+import styled from 'styled-components';
+import { Icon, variables } from '@trezor/components';
+import { Translation } from '@suite-components';
+import { useSelector } from '@suite-hooks/useSelector';
+import { selectCurrentTargetAnonymity } from '@wallet-reducers/coinjoinReducer';
+import { darken } from 'polished';
+import { defaultColorVariant } from '@trezor/theme';
+
+const Container = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 6px 10px;
+    border-radius: 8px;
+
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    font-size: ${variables.FONT_SIZE.SMALL};
+    text-align: right;
+
+    background: ${({ theme }) => theme.STROKE_GREY};
+
+    &:hover,
+    &:focus,
+    &:active {
+        background: ${({ theme }) => darken(theme.HOVER_DARKEN_FILTER, theme.STROKE_GREY)};
+    }
+
+    cursor: ${({ onClick }) => (onClick ? 'pointer' : 'default')};
+`;
+
+const Values = styled.div`
+    min-width: 64px;
+`;
+
+const AnonymityLevel = styled.p`
+    font-variant-numeric: tabular-nums;
+`;
+
+const AnonymityStatus = styled.p<{ color: string }>`
+    color: ${({ color }) => color};
+    font-size: ${variables.FONT_SIZE.TINY};
+`;
+
+const anonymityStatus = {
+    bad: {
```

Comment:
```suggestion
    [AnonymityLevel.Bad]: {
```

I would create an enum.

```suggestion
enum AnonymityLevel {
    Bad,
    Medium,...
}
```

Reply:
I don't see any benefit from enum here, but I at least mark the dictionary object as const. Is that enough? or would you still prefer enum?

Reply:
check now, please, refactored